pkg/test-infra/pkg/tidb/recommend.go

```diff
@@ -18,15 +18,18 @@ import (
 	"strings"
 
 	"github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/pingcap/tipocket/pkg/test-infra/pkg/fixture"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type TiDBClusterRecommendation struct {
 	*v1alpha1.TidbCluster
 	*corev1.Service
+	NS   string
+	Name string
```

Comment:
NS and Name are same as TidbCluster.ObjectMeta.Namespace and TidbCluster.ObjectMeta.Name, so I think we can remove these two new fields. What's your opinion?

Reply:
In binlog cluster, two tidb cluster with different names are in the same namespace.

https://github.com/pingcap/tipocket/blob/005f3f1824075707165fbc32dcf1da671c2f4f79/pkg/test-infra/pkg/binlog/recommander.go#L49-L50